src/components/PathwayExercise/ContentEdit/index.js

```diff
@@ -0,0 +1,487 @@
+      setCourse(temp);
+    }
+    // if (ap === "output") {
+    //   const AddHeader = { ...allComponents.header.AddHeader1 };
+    //   console.log("temp before", temp);
+    //   temp.splice(index, 0, AddHeader);
+    //   console.log("temp after", temp);
+    //   setCourse(temp);
+    // }
+  };
+
+  const putApiAssessmentCall = () => {
+    //delete the component which value is empty string or empty list
+    const tempCourse = [];
+    for (var i in course) {
+      if (course[i].value.length !== 0) {
+        tempCourse.push(course[i]);
+      }
+    }
+    const stringifiedCourse = JSON.stringify(tempCourse, null, 0);
+    console.log(id, stringifiedCourse, "cc");
+    axios({
+      method: METHODS.PUT,
+      url: `${process.env.REACT_APP_MERAKI_URL}/assessment/${id}`,
+      headers: {
+        "version-code": versionCode,
+        accept: "application/json",
+        Authorization: user.data?.token || "",
+      },
+      data: {
+        content: stringifiedCourse,
+      },
+    }).then((res) => {
+      history.push(
+        interpolatePath(PATHS.PATHWAY_COURSE_CONTENT, {
+          courseId: params.courseId,
+          exerciseId: params.exerciseId,
+          pathwayId: params.pathwayId,
+        })
+      );
+      console.log(res, "res");
+    });
+  };
+
+  useEffect(() => {
+    console.log("Punnu");
+    axios({
+      method: METHODS.GET,
+      url: `${process.env.REACT_APP_MERAKI_URL}/courses/${courseId}/exercises`,
+      headers: {
+        "version-code": versionCode,
+        accept: "application/json",
+        Authorization: user.data?.token || "",
+      },
+    })
+      .then((res) => {
+        console.log("res", res);
+        const course_type = res.data.course.exercises[exerciseId].content_type;
+        setCourseType(course_type);
+        if (course_type === "assessment") {
+          if (
+            res.data.course.exercises[exerciseId].content[0].component !==
+            "questionExpression"
+          ) {
+            res.data.course.exercises[exerciseId].content.splice(0, 0, {
+              component: "questionExpression",
+              type: "python",
+              title: "",
+              value: "",
+            });
+          }
+          if (
+            res.data.course.exercises[exerciseId].content[1].component !==
+            "questionCode"
+          ) {
+            res.data.course.exercises[exerciseId].content.splice(1, 0, {
+              component: "questionCode",
+              type: "python",
+              title: "",
+              value: "",
+            });
+          }
+        }
+        setId(res.data.course.exercises[exerciseId].id);
+        setCourse(res.data.course.exercises[exerciseId].content);
+        const updated_at = res.data.course.exercises[exerciseId].updated_at;
+        const longDateStr = formatInUtc(updated_at, "EEEE dd MMMM yyyy");
+        // console.log("longDateStr", longDateStr);
+        setUpdatedOn(longDateStr);
```

Comment:
As noted in https://github.com/navgurukul/bhanwari-devi/pull/804#issuecomment-1321073712 and https://github.com/navgurukul/bhanwari-devi/pull/804#issuecomment-1321080007, you should be able to use `format` here instead. Also, we don't need the extra variable declaration since it's used immediately and not used anywhere else.
```suggestion
        setUpdatedOn(format(updated_at, "EEEE dd MMMM yyyy"));
```